### PR TITLE
cannon: remove old test-util preimage-cheat function

### PR DESCRIPTION
**Pull request by protolambda**
**Description**

Remove the old `cheat` function from the pre-image oracle, to prepare it for secure usage onchain.

This util, in older versions of cannon, was used for testing purposes: to load the local game-specific preimages, without needing a localized context.

The `loadLocalData` replaced it, by hashing the key with the caller, which then attaches it to the dispute-game instance, through `addLocalData`, while preventing arbitrary data inputs.

### CODE REVIEW

File: packages/contracts-bedrock/.gas-snapshot [hunk 1]
```diff
@@ -17,7 +17,7 @@ AttestationStationTest:test_attest_bulk_succeeds() (gas: 703749)
 AttestationStationTest:test_attest_individual_succeeds() (gas: 632087)
 AttestationStationTest:test_attest_single_succeeds() (gas: 651325)
 BlockOracle_Test:test_checkpointAndLoad_succeeds() (gas: 58341)
-BlockOracle_Test:test_load_noBlockHash_reverts() (gas: 12805)
+BlockOracle_Test:test_load_noBlockHash_reverts() (gas: 13659)
```
Comment (protolambda) [inline packages/contracts-bedrock/.gas-snapshot]: tried updating the gas snapshot, but the gas changes make no sense to me. Is there a specific check I can do to fix this?

Comment (tynes) [reply packages/contracts-bedrock/.gas-snapshot]: I am guessing you are using the wrong version of forge

Comment (protolambda) [reply packages/contracts-bedrock/.gas-snapshot]: Can conffirm, my bad. Running `pnpm update:foundry` worked

**Review by Inphi — APPROVED** [review]
Thank you!

**Pull request merged**